src/program.ts

```diff
@@ -3639,6 +3639,10 @@ export class Local extends VariableLikeElement {
     assert(type != Type.void);
     this.setType(type);
   }
+
+  isInFunction(fn: Function): bool {
```

Comment:
I liked the original signature that you gave it (the one with the `Flow` parameter), because now this is just comparing a field to a given Function object. If you agree, maybe `isInFlow` or `declaredByFlow` would be a better name.